test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -150,15 +147,18 @@ func TestSchedule1000Node30KPods(t *testing.T) {
 // 	}
 // }
 
+// testConfig contains the some input parameters needed for running test-suite
 type testConfig struct {
-	numPods                   int
-	numNodes                  int
+	// Note: We don't need numPods, numNodes anymore in this struct but keeping them for backward compatibility
```

Comment:
Where numNodes got replaced, i.e. how you set number of Nodes created? From what I can tell it's still used by the `mutate` function.